.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
 __pycache__
 .pytest_cache
-src/tempCodeRunnerFile.py
\ No newline at end of file
+src/tempCodeRunnerFile.py
+.DS_Store
```

Comment:
what are these 2 here?

Reply:
They are clutter files. tempCodeRunnerFile is created by vscode during execution, DS_Store is used for custom attributes.